src/config.py

```diff
@@ -22,6 +22,14 @@
 
 ALL_RELEASE_TYPES = [MAJOR_RELEASE, MINOR_RELEASE, PATCH_RELEASE]
 
+BRANCH_SUFFIX_RANDOM = "random"
+BRANCH_SUFFIX_TIMESTAMP = "timestamp"
+
+BRANCH_SUFFIXES = [
+    BRANCH_SUFFIX_RANDOM,
+    BRANCH_SUFFIX_TIMESTAMP,
+]
```

Comment:
I am not sure what use cases users will have to choose one over the other. The suffix was added to create a unique branch. Maybe we can omit this option?

Reply:
You are right. Thanks for the review.

I took the API idea from https://github.com/peter-evans/create-pull-request. I implement only the first two because the `short-commit-hash` needs to rework the entire flow... It may be too configurable.

I will stick to a configuration for the branch name and another option to enable/disable uniqueness. With a default behavior as before.

You are also right, I need to check the pull request status before forcing the push.

Reply:
Updated but I still need to handle existing pull request.

Reply:
I have updated the PR. The way I have handled the existing PR is the same as here:
https://github.com/peter-evans/create-pull-request/blob/main/src/github-helper.ts#L70

The code seems to work but I have no way to test it. I recommend you to do a minimum of testing before merging.